block.json

```
@@ -67,5 +67,6 @@
  			"default": []
  		}
  	},
-	"editorScript": "file:build/index.js"
+	"editorScript": "file:build/index.js",
```

instead of registering the file here, we are also able to pass it the handle with which we have registered the stript. So `maps-block-apple-block` in this case
```suggestion
	"editorScript": "maps-block-apple-block",
```